scheduler/scheduler.yml

```diff
@@ -41,12 +41,17 @@
 #   ansible.builtin.import_playbook: ../utils/update_user_repo.yml
 #   when: not ( hostvars['127.0.0.1']['update_user_repo_executed'] | default(false) | bool )
 
-- name: Apply common Slurm installation and config
-  hosts: slurm_control_node, slurm_node, login
-  gather_facts: false
+- name: Install Slurm
+  hosts: slurm_control_node, slurm_node, slurm_dbd, login
   any_errors_fatal: true
+  pre_tasks:
+    - name: Include input project directory
+      ansible.builtin.import_role:
```

Comment:
we are already running this role in line 22. we dont need to run this utility again